### DISCUSSION TITLE
Share your PeakRDL plugins here

### DISCUSSION THREAD

**Discussion by amykyta3** [SystemRDL/systemrdl-compiler#157] [Show and tell]
I started a community plugin page.
If you've made something neat you'd like to share, feel free to submit an addition!

https://github.com/SystemRDL/.github/blob/main/community_plugins.md

**Comment by Risto97** [comment]
OpenTitan project has a similar tool to PeakRDL toolchain.
The format they are using is a hjson file format and the description can be found here: https://opentitan.org/book/util/reggen/index.html.
While the formats might not be completely compatible, they can still be converted between each other at a limited subset.

I needed to use some OpenTitan peripherals and wanted to integrate them into my project which is mostly SystemRDL based, to keep the documentation, CPP drivers consistent, etc...

I created a opentitan exporter, and importer. 
Still a work in progress, but it can work for my use case. I will be updating it to cover more features.
- Source: https://github.com/Risto97/PeakRDL-opentitan
- Website: https://risto97.github.io/PeakRDL-opentitan/index.html
